src/OrchardCore.Modules/OrchardCore.ContentPreview/Views/Preview/Index.cshtml

```diff
@@ -28,14 +28,6 @@
     <ul></ul>
 </div>
 
-<div id="serverErrorWarning" class="alert alert-warning alert-dismissible fade show" role="alert" style="position:fixed; top:10px;right:40px; display:none; z-index:999999;">
```

Comment:
Not sure why this is removed, it was showing error messages on the front end, we merged it yesterday.

Reply:
@sebastienros oh no, view the whole file, it is still there. It shows up here because i integrated it before merging dev, this because i added to the close button an `id="close-server-warning"` this to be able to hide it, as already done for the `notConnectedWarning` whose i changed the close button id to `id="close-connect-warning"` to differentiate it.